item-seller/readme.md

```diff
@@ -1,4 +1,15 @@
-This guide will walk you through the process of building contracts, deploying them into an existing world running in Docker, and testing their functionality by executing scripts.
+## Introduction
+This guide will walk you through the process of building contracts for the item seller, deploying them into an existing world running in Docker, and testing their functionality by executing scripts. The contracts will be used with a SSU (Smart Storage Unit) to act as a item seller where players can purchase items, for example lenses for ERC20 Tokens.
+
+### User Process
+#### Step 0: User buys an item
```

Comment:
Can we start the step 0 when we start with the instructions ? Here in this section we can just write about the user Flow ?

Reply:
Good point. Have sorted.

Reply:
It will be nice to include a diagram too. Nice to have not priority